spynnaker/pyNN/models/spike_source/spike_source_poisson_variable.py

```diff
@@ -0,0 +1,34 @@
+from .spike_source_poisson_vertex import SpikeSourcePoissonVertex
+from spynnaker.pyNN.models.abstract_pynn_model import AbstractPyNNModel
+
+_population_parameters = {"seed": None}
+
+DEFAULT_MAX_ATOMS_PER_CORE = 500
+
+
+class SpikeSourcePoissonVariable(AbstractPyNNModel):
+
+    default_population_parameters = _population_parameters
+
+    def __init__(self, rates=[1.0], starts=[0], durations=None):
```

Comment:
Alas, this is Bad Practice in Python because it is using a read-write parameter as a default. Can we use a `tuple` instead? (i.e., `tuple(1.0)` and `tuple(0)`)

Reply:
I suspect these should be required parameters instead - surely you always want to supply a list of rates and start times...